src/components/ValueCurrency.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2021 Robbert Broersma
+ */
+
+import clsx from "clsx";
+import { ForwardedRef, forwardRef, HTMLAttributes } from "react";
+
+interface ValueCurrencyProps extends HTMLAttributes<HTMLDataElement> {
+  currency: string;
+  amount: string | number;
+  locale: string;
```

Comment:
Suggestion: make this optional, and "nl_NL" by default?

```suggestion
  locale?: string;
```